### ISSUE TITLE
When i click to "Retry" programm to close

### ISSUE THREAD

**Issue by pushkinb** [shaked6540/YoutubePlaylistDownloader#399]
And some tracks don't downloads if even programm don't have a errors

**Comment by pushkinb** [comment]
![image](https://github.com/user-attachments/assets/6dc834ce-2f0f-4b2c-95db-bc91bf344ad7)

**Comment by pushkinb** [comment]
and I also wanted to say that the program does not download 3 videos, but writes the name of only one not downloaded in a text file, so I can't download them manually

**Comment by pushkinb** [comment]
![image](https://github.com/user-attachments/assets/2754df14-d5ef-4dd2-877e-a86812f8f5c2)

**Comment by shaked6540** [comment]
Can you provide links to the videos that did not download?

**Comment by pushkinb** [comment]
Yes, https://www.youtube.com/watch?v=9RysoTLmLOk&list=PLD1CD348302D621E5&index=29&pp=iAQB8AUB

**Comment by shaked6540** [comment]
Thanks, I see this problem on my pc as well. I will try to find out whats wrong, it will take me some time. 

Regarding the issue with the Retry button causing the program to crash, I am checking it out as well.

**Comment by pushkinb** [comment]
Thank you for your efforts, I will be waiting!

**Comment by pushkinb** [comment]
I also noticed that the downloaded file differs in duration from that video on YouTube, mostly by 1-2 minutes the downloaded file lasts longer, and this happens if I turn on the automatic conversion of the downloaded file to mp3

**Comment by shaked6540** [comment]
Maybe you quit the program while the conversion is still happening? this really shouldn't happen, and its the first time im hearing about it
